### ISSUE TITLE
"nan" values 

### ISSUE THREAD

**Issue by Jes46** [ultralytics/ultralytics#7277]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Train

### Bug

Hello, I'm trying to train my own dataset with YOLOv8 and all times I've tried I have the following result wich made my model can't able to work. In all parameters I get "nan" value.
This is an screenshot and the two lines code where I made the training.

![Captura2](https://github.com/ultralytics/ultralytics/assets/150959706/6a5a7193-7c16-46f1-9008-0d3e8c403717)

![Captura](https://github.com/ultralytics/ultralytics/assets/150959706/a1a42665-e215-4629-b419-e47b59600cc1)

Thanks.

### Environment

I'm using Visual Studio Code with Python 3.9.12 64 bits

### Minimal Reproducible Example

Here you can see what is happening with this issue in my model.

![results](https://github.com/ultralytics/ultralytics/assets/150959706/abda12d0-cd0a-43b8-832b-710db4a95a00)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by Kayzwer** [comment]
try to reduce lr, and also set warmup_epochs to 0

**Comment by sunmooncode** [comment]
I have encountered a similar situation, and the reason for my mistake was not adding the corresponding label files to the background data.

**Comment by Jes46** [comment]
Thanks so much for your comment. I tried to see if my data was wrong but they were right. I could fix it changing the hyperparametrer "amp" from True to False because is something that affect GPU (I don't know exatly).

**code line: model.train(data=data_path, epochs=30, batch=4, amp=False)**

I hope this will help you as well in the future if you need it.

**Comment by glenn-jocher** [comment]
@Jes46 great to hear you resolved the issue by adjusting the `amp` parameter! Automatic Mixed Precision (AMP) can indeed affect training stability on some systems, especially if the GPU doesn't fully support it. Your experience is valuable, and it's a good reminder that hyperparameter tuning can be crucial for successful model training. Keep experimenting and happy training! 😊🚀

**Comment by simusid** [comment]
3 March - I've just installed the latest build and I was having the same "NaN" issue.   Setting amp=False does give me valid results.
